devops/cookiecutter-girder-plugin/{{ cookiecutter.plugin_name }}/web_client/views/ConfigView.js

```diff
@@ -0,0 +1,77 @@
+import _ from 'underscore';
+
+import PluginConfigBreadcrumbWidget from 'girder/views/widgets/PluginConfigBreadcrumbWidget';
+import View from 'girder/views/View';
+import events from 'girder/events';
+import { restRequest } from 'girder/rest';
+
+import ConfigViewTemplate from '../templates/configView.pug';
+
+{% set plugin_name_dashed = cookiecutter.plugin_name|replace("_", "-") -%}
+var ConfigView = View.extend({
+    events: {
+        'submit #{{ plugin_name_dashed }}-settings-form': function (event) {
+            event.preventDefault();
+            this.$('#{{ plugin_name_dashed }}-settings-error-message').empty();
+
+            this._saveSettings([{
+                key: '{{ cookiecutter.plugin_name }}.item_metadata',
+                value: this.$('#{{ plugin_name_dashed }}-item-metadata').val().trim()
+            }]);
+        }
+    },
+
+    initialize: function () {
+        restRequest({
+            type: 'GET',
+            path: 'system/setting',
+            data: {
+                list: JSON.stringify([
+                    '{{ cookiecutter.plugin_name }}.item_metadata'
+                ])
+            }
+        }).done(_.bind(function (resp) {
+            this.render();
+            this.$('#{{ plugin_name_dashed}}-item-metadata').val(resp['{{ cookiecutter.plugin_name }}.item_metadata']);
+        }, this));
+    },
+
+    render: function () {
+        this.$el.html(ConfigViewTemplate());
+
+        if (!this.breadcrumb) {
+            this.breadcrumb = new PluginConfigBreadcrumbWidget({
+                pluginName: '{{ cookiecutter.plugin_nice_name }}',
+                el: this.$('.g-config-breadcrumb-container'),
+                parentView: this
+            });
+        }
```

Comment:
I would just construct the breadcrumb view in `initialize` unless there is a specific reason to defer it until the first render.